torch/distributions/gumbel.py

```diff
@@ -24,6 +24,7 @@ class Gumbel(Distribution):
     has_rsample = True
     params = {'loc': constraints.real, 'scale': constraints.positive}
     support = constraints.real
+    _euler_constant = 0.57721566490153286060  # Euler Mascheroni Constant
```

Comment:
nit: I would make this a file-level constant rather than a class constant. Is there a reason you've added this to the class?

Reply:
I wasn't sure if it would be called without problem from other places if it was file level.